base/applications/taskmgr/taskmgr.h

```diff
@@ -15,7 +15,7 @@
 
 #define STATUS_WINDOW	2001
 #define STATUS_SIZE1	85
-#define STATUS_SIZE2	190
+#define STATUS_SIZE2	157 // he-IL.rc determines minimum width:  72 == 157 - 85
```

Comment:
```suggestion
#define STATUS_SIZE2	157 // he-IL.rc determines minimum width: 72 == 157 - 85
```
now reactosfanboy-nimize the hermification